### PR TITLE
Add overloaded "isEnabled" method that takes loggerClassName instead of callerDepth (#381)

### PR DESCRIPTION
**Pull request by q3769**
### Description

Linked issue:

#381

### CODE REVIEW

File: tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java [hunk 1]
```diff
@@ -122,6 +123,36 @@ public void isEnabled() {
 		verify(second, atLeastOnce()).isEnabled(eq(2), isNull(), any());
 	}
 
+	/**
+	 * Verifies that {@code isEnabled} method evaluates the severity level from underlying logging providers and
```
Comment (pmwmedia) [inline tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java]: {@code isEnabled} method -> {@code isEnabled()}

File: tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java [hunk 2]
```diff
@@ -122,6 +123,36 @@ public void isEnabled() {
 		verify(second, atLeastOnce()).isEnabled(eq(2), isNull(), any());
 	}
 
+	/**
+	 * Verifies that {@code isEnabled} method evaluates the severity level from underlying logging providers and
+	 * returns {@code true} if given severity level is enabled at least for one of the underlying logging providers.
```
Comment (pmwmedia) [inline tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java]: if given severity level ->  if the given severity level

File: tinylog-api/src/test/java/org/tinylog/provider/NopLoggingProviderTest.java [hunk 3]
```diff
@@ -69,6 +69,12 @@ public void isEnabledUntagged() {
 		assertThat(provider.isEnabled(2, null, Level.INFO)).isFalse();
 		assertThat(provider.isEnabled(3, null, Level.WARN)).isFalse();
 		assertThat(provider.isEnabled(4, null, Level.ERROR)).isFalse();
+
+		assertThat(provider.isEnabled("anyClassName", null, Level.TRACE)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", null, Level.DEBUG)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", null, Level.INFO)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", null, Level.WARN)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", null, Level.ERROR)).isFalse();
```
Comment (pmwmedia) [inline tinylog-api/src/test/java/org/tinylog/provider/NopLoggingProviderTest.java]: "anyClassName" -> "org.foo.Bar" or any other fully-qualified dummy class name

File: tinylog-api/src/test/java/org/tinylog/provider/NopLoggingProviderTest.java [hunk 4]
```diff
@@ -81,6 +87,12 @@ public void isEnabledTagged() {
 		assertThat(provider.isEnabled(2, "test", Level.INFO)).isFalse();
 		assertThat(provider.isEnabled(3, "test", Level.WARN)).isFalse();
 		assertThat(provider.isEnabled(4, "test", Level.ERROR)).isFalse();
+
+		assertThat(provider.isEnabled("anyClassName", "test", Level.TRACE)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", "test", Level.DEBUG)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", "test", Level.INFO)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", "test", Level.WARN)).isFalse();
+		assertThat(provider.isEnabled("anyClassName", "test", Level.ERROR)).isFalse();
```
Comment (pmwmedia) [inline tinylog-api/src/test/java/org/tinylog/provider/NopLoggingProviderTest.java]: "anyClassName" -> "org.foo.Bar" or any other fully-qualified dummy class name

**Review by pmwmedia — CHANGES_REQUESTED** [review]

File: tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java [hunk 5]
```diff
@@ -124,8 +124,8 @@ public void isEnabled() {
 	}
 
 	/**
-	 * Verifies that {@code isEnabled} method evaluates the severity level from underlying logging providers and
-	 * returns {@code true} if given severity level is enabled at least for one of the underlying logging providers.
+	 * Verifies that {@code isEnabled()} method evaluates the severity level from underlying logging providers and
```
Comment (pmwmedia) [inline tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java]: Please remove the word "method". It is redundant.

**Review by pmwmedia — CHANGES_REQUESTED** [review]

File: tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java (hunk 5 above)
Comment (q3769) [reply tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java]: I removed it. However, the Javadoc was copied from previous ones where there is a consistent pattern of using the word "method". I did not try to change those similar places as part of this PR.

Comment (pmwmedia) [reply tinylog-api/src/test/java/org/tinylog/provider/BundleLoggingProviderTest.java]: You are absolutely right about the consistency. Unfortunately, the Javadoc of the other method is not written in good English.

**Review by pmwmedia — APPROVED** [review]
Thank you for your great PR!

**Pull request merged**